Libs/vtkAddon/CMakeLists.txt

```diff
@@ -64,6 +64,14 @@ set(vtkAddon_SRCS
   vtkAddonMathUtilities.h
   vtkAddonMathUtilities.cxx
   vtkAddonSetGet.h
+  vtkStreamingVolumeCodec.cxx
+  vtkStreamingVolumeCodec.h
+  vtkStreamingVolumeFrame.cxx
+  vtkStreamingVolumeFrame.h
+  vtkStreamingVolumeCodecFactory.cxx
+  vtkStreamingVolumeCodecFactory.h
+  vtkRGBVolumeCodec.cxx
```

Comment:
It looks like some actual codec. It would be better to include uncompressed, raw, trivial, simple, plain or some similar word in the name.